library/tasks/SplitBamByCellBarcode.wdl

```diff
@@ -6,13 +6,14 @@ task SplitBamByCellBarcode {
     Float size_in_mb = 1024.0
 
     # runtime values
-    String docker = "quay.io/humancellatlas/secondary-analysis-sctools:v0.3.5"
+    String docker = "quay.io/humancellatlas/secondary-analysis-sctools:v0.3.9"
 
     Int machine_mem_mb = 15258
     Int cpu = 16
 
     # we can calculate disk size for arrays of input files in WDL 1.0
-    Int disk = 500
+    Int disk = 3500
```

Comment:
should we calculate this based on input size rather than just always getting a ton of disk?